Easydict/Feature/Configuration/Configuration.swift

```diff
@@ -140,7 +140,8 @@ class Configuration: NSObject {
     }
 
     var fontSizeRatio: CGFloat {
-        fontSizes[Int(fontSizeIndex)]
+        let safeIndex = max(0, min(Int(fontSizeIndex), fontSizes.count - 1))
```

Comment:
Add a safety guard against having store dirty value that is out of array bounds.